app/assets/javascripts/conference_form.js

```diff
@@ -0,0 +1,23 @@
+$(function () {
+  $('#new_conference').on('ajax:success',
```

Comment:
Personal preference, but what do you think about using a data attribute instead of the ID for attaching the behavior to this element? It could be something like `data-behavior="new-conference-form"` 🤔 
I personally prefer to leave classes and ids for styling elements and use the more powerful (you can pass down data :v:) data attributes for adding certain behavior.
What do you think?

Reply:
I can do this, @klappradla. No problem!